pkg/bot/discord.go

```diff
@@ -0,0 +1,134 @@
+package bot
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/infracloudio/botkube/pkg/config"
+	"github.com/infracloudio/botkube/pkg/execute"
+	"github.com/infracloudio/botkube/pkg/logging"
+)
+
+// DiscordBot listens for user's message, execute commands and sends back the response
+type DiscordBot struct {
+	Token          string
+	AllowKubectl   bool
+	RestrictAccess bool
+	ClusterName    string
+	ChannelID      string
+	SlackURL       string
```

Comment:
Wonder why this is named SlackURL ? Not sure if we need restructring across other providers as well to fix.
cc @PrasadG193

Reply:
removed unused variables.